src/components/PopupWithConfirmation.js

```diff
@@ -0,0 +1,26 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithConfirmation extends Popup {
+  constructor(popup, submitButton) {
+    super(popup, submitButton);
+    this._confirmButton = document.querySelector(submitButton);
+  }
+
+  open() {
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.